src/scripts/main.js

```diff
@@ -1,3 +1,214 @@
+    let countActive = 0;
+
+    for (const r of tbodyRows) {
+      if (r.classList.contains('active')) {
+        countActive++;
+      }
+    }
+
+    row.classList.toggle('active', countActive < 1);
+  }
+});
+
+const body = document.body;
+const form = document.createElement('form');
+
+// FORM
+
+form.classList.add('new-employee-form');
+
+form.innerHTML = `
+<label>Name: <input name='name' type='text' data-qa="name" required></label>
+<label>Position: <input name='position' type='text' data-qa="position" required></label>
+<label>Office:
+  <select name="office" id="office" data-qa="office" required>
+    <option value="tokyo">Tokyo</option>
+    <option value="singapore">Singapore</option>
+    <option value="london">London</option>
+    <option value="new york">New York</option>
+    <option value="edinburgh">Edinburgh</option>
+    <option value="san francisco">San Francisco</option>
+  </select>
+</label>
+<label>Age: <input name='age' type='number' data-qa="age" required> </label>
+<label>Salary: <input name='salary' type='number' data-qa="salary" required> </label>
+<button type='submit'>Save to table</button>
+`;
+body.appendChild(form);
+
+// BUTTON
+
+const button = body.querySelector("[type='submit']");
+const createRow = (data) => {
+  const row = document.createElement('tr');
+
+  const convertToString = (num) => {
+    if (num === null || num === undefined || isNaN(+num)) {
+      return '';
+    }
+
+    return `$${(+num).toLocaleString('en-US')}`;
+  };
+
+  row.innerHTML = `
+    <td>${data.name}</td>
+    <td>${data.position}</td>
+    <td>${data.office}</td>
+    <td>${data.age}</td>
+    <td>${convertToString(data.salary)}</td>
+  `;
+
+  tbody.appendChild(row);
+  tbody = table.tBodies[0];
+  tbodyRows = [...tbody.rows];
+};
+// NOTIFICATION
+const pushNotification = (posRight, title, description, type) => {
+  const posTop = body.offsetHeight / 2 - body.getBoundingClientRect().top;
+  const notification = document.createElement('div');
+  const h2 = document.createElement('h2');
+  const p = document.createElement('p');
+
+  notification.classList.add('notification');
+  notification.classList.add(type);
+  notification.setAttribute('data-qa', 'notification');
+  notification.style.top = `${posTop}px`;
+  notification.style.right = `${posRight}px`;
+
+  h2.textContent = title;
+  p.textContent = description;
+  notification.append(h2, p);
+
+  body.appendChild(notification);
+  setTimeout(() => (notification.style.display = 'none'), 2000);
+};
+
+button.addEventListener('click', (e) => {
+  e.preventDefault();
+
+  const formData = new FormData(form);
+  const data = Object.fromEntries(formData.entries());
+
+  const isValide = () => {
+    const checkInput = Object.values(data).every((val) => val !== '');
+    const checkName = data.name.length >= 4;
+    const checkAge = data.age >= 18 && data.age <= 90;
+
+    if (checkInput && checkName && checkAge) {
+      return true;
+    }
+
+    return false;
```

Comment:
The `isValide` function does not validate the salary field properly. Ensure that the salary is a positive number and formatted correctly. Consider adding a check for the salary field similar to the age validation.